api/src/main/resources/workbench.yaml

```diff
@@ -993,21 +995,13 @@ paths:
           required: false
           description: Sort direction (default is ascending)
         - in: query
-          name: filterColumn
+          name: filters
```

Comment:
IMHO it would be better to have our API define a Filter model object, and use it here and above. We should only pass around strings for stuff like this when we have a strong reason to.

(One could argue the same thing about cohort definitions, though I'm not suggesting we change that right now.)

Reply:
I agree, but I don't think that we can define a model object when using an query parameter in swagger 2.0. I could have missed it in the documentation.

Reply:
Right, so this would mean switching to define a request object instead of using query params.

If that's too painful, I will withdraw my objection. :)

Reply:
I can switch it to be a post. It will take some rework but shouldn't change the underlying logic that much.